app/scripts/router.js

```diff
@@ -317,6 +319,10 @@ function (
       this.navigate(url);
     },
 
+    getCurrentPage: function () {
+      return this._currentPage || Backbone.history.fragment;
```

Comment:
If `Backbone.history.fragment` always returns the correct value, no reason to keep track of it locally and introduce a second source of truth.